src/com/tekion/cricket/Main.java

```diff
@@ -0,0 +1,28 @@
+package com.tekion.cricket;
+
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args)
+    {
+        int choice;
+        System.out.println("Press 1 to start a Match , Press 2 to start a series");
+        Scanner sc =  new Scanner(System.in);
+        choice = sc.nextInt();
+
+        switch(choice){
+            case 1:
```

Comment:
should we have enums maintaining these 1s and 2s into something like ONE & TWO ?

Reply:
sure. I'll do that.